authenticate.go

```diff
@@ -155,6 +162,52 @@ func (c *OktaClient) handleMFAChallenge(transaction api.AuthenticationTransactio
 	case factors.FactorTypeU2F:
 		return c.handleFactorTypeU2F(transaction)
 
+	case factors.FactorTypeWebAuthN:
+		profile, ok := transaction.Embedded.Factor.Profile.(api.FactorProfileWebAuthN)
+		if !ok {
+			c.log("Profile was not of type FactorProfileWebAuthN: %s", transaction.Embedded.Factor.Profile)
+			return c.cancelCurrentFactorWithErrorMessage(transaction, unexpectedErrorMessage)
+		}
+
+		// Setup a context with the timeout set to the value provided by Okta
+		timeoutSeconds := 30
+		ctx, _ := context.WithTimeout(context.Background(), time.Second*time.Duration(timeoutSeconds))
+
+		authResp, err := c.prompts.VerifyU2F(ctx, VerifyU2FRequest{
```

Comment:
Where is `VerifyU2FRequest` implemented? I'm trying to get webauthn with Okta to work in Go with code very similar to https://github.com/segmentio/aws-okta/pull/201 using https://github.com/marshallbrekka/go-u2fhost under the hood. I'm getting a "The provided key handle is not present on the device, or was created with a different application parameter" error.

If you have a Go implementation of U2F that works with the new Okta webauthn factors, I'd love to know about it!

I've narrowed it down to